neural_compressor/torch/algorithms/layer_wise/torch_load.py

```diff
@@ -0,0 +1,272 @@
+#!/usr/bin/env python
```

Comment:
The file located in the torch folder does not require the `torch_` prefix; it would be better to rename it to `load.py`.

Reply:
ok, the layer_wise is not refined in this PR. It will come in the next PR of layer-wise support.